src/utils/authFetch.js

```diff
@@ -6,7 +6,8 @@ export const authFetch = (url, data) => {
     const dataHeaders = data?.headers
     const authHeaders = authToken ? {[loginAPI.jwtTokenKey]: authToken} : {}
     const newHeaders = dataHeaders ? Object.assign(dataHeaders, authHeaders) : authHeaders
-    const authData = Object.assign(data | {}, {headers: newHeaders})
+    const authData = Object.assign(data || {}, {headers: newHeaders})
+    console.log(url, authData)
```

Comment:
Po pierwsze console.log, a po drugie czym się różni w jsie | od ||?

Reply:
Szczerze mówiąc, nie wiem, czy pierwsze w ogóle istnieje, czy nie była to literówka Kuby. Jak zawsze widziałem, to używali ||, jeśli to po lewej stronie może być null / undefined. 
https://stackoverflow.com/questions/4576867/javascript-or-operator-with-a-undefinded-variable -> tu masz stare odpowiedzi, jak to działa, ale dalej to tak działa.

Sprawdziłem, | oznacza binarnego ora, ale nie mam zielonego pojęcia, jaki byłby wynik data binarny or pusty obiekt. Możecie mi zabrać moje cztery z WDI, ale ni ciula tego nie wymyślę XDDDD